src/providers/bitbucket.ts

```diff
@@ -0,0 +1,40 @@
+import { OAuthConfig, OAuthUserConfig } from "./oauth"
+
+interface BitbucketProfile {
```

Comment:
I see these fields are the ones that are being used by default in the `profile` callback, but if you look at the logs, is there anything else being returned from Bitbucket? Those could be added here as well for convenience to the user.

Reply:
Sure, I will follow this resource: [Bitbucket API](https://developer.atlassian.com/bitbucket/api/2/reference/resource/user)